js/src/forum/Components/CalendarPage.js

```diff
@@ -54,75 +51,40 @@ export default class CalendarPage extends Page {
    */
   sidebarItems() {
     const items = IndexPage.prototype.sidebarItems();
-    //new evemt
+    // newDiscussion = newEvent
     if(app.session.user){
       if(app.session.user.canStartEvents()) {
         items.replace('newDiscussion',
           Button.component({
-            children: app.translator.trans('flarum-calendar.forum.button.create'),
             icon: 'fas fa-calendar-plus',
             className: 'Button Button--primary PollModal-SubmitButton',
             itemClassName: 'App-primaryControl',
             onclick: this.openCreateModal.bind(this)
-          })
+          }, app.translator.trans('flarum-calendar.forum.button.create'))
         );
-      }else{
-        items.remove('newDiscussion');
-      }
+      }else{ items.remove('newDiscussion'); }
```

Comment:
Let's keep this on separate lines like before and below (124-126, etc)  I have always preferred it's readability.

Reply:
^^